### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by SabulineHorizon**
Added Gazillionaire autosplitter

If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -13821,3 +13821,15 @@
         <Website>https://github.com/Marek2810/Vlak</Website>
     </AutoSplitter>
 </AutoSplitters>
```
Comment (NoTeefy) [inline LiveSplit.AutoSplitters.xml:13823]: @SabulineHorizon You need to place your <AutoSplitter></AutoSplitter> section before the closing </AutoSplitters> Tag.

Comment (SabulineHorizon) [reply LiveSplit.AutoSplitters.xml]: I have now edited it. I think the tag should be in the correct position now.

Comment (SabulineHorizon) [reply LiveSplit.AutoSplitters.xml]: I just now realized that you made the change and that I could have simply confirmed it. Should I go ahead and confirm your edit and cancel my last one, or leave it as is? I'm new to GitHub, so I don't know what the standard procedures are yet.

Comment (NoTeefy) [reply LiveSplit.AutoSplitters.xml]: Nah, I didn't change the code. It was only asking you to accept my review or not 😅

**Pull request merged**